### PR TITLE
My webpage version 1.0

### PR DESCRIPTION
**Pull request by ricglz**
I create my webpage using html, css and bootstrap.

### CODE REVIEW

**Comment by AlexHenkel** [comment]
Congratulations! It's a great start. But, what about including Bootstrap in your project? I think it will improve a lot your design in order to be responsive out of the box

**Comment by ricglz0201** [comment]
Oh! I thought I had included it, I will add it.

File: Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html [hunk 1]
```diff
@@ -1,133 +1,142 @@
...
+                            <tr>
+                                <td>San Francisco</td>
+                                <td>HTML and CSS</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+                <div class="Sign-Up">
+                    <h1>Sign Up!</h1>
+                    <form>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html]: You should only use one `<form>` tag that wraps all your inputs, selects or any other components of the form plus the submit button.

File: Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html [hunk 2]
```diff
@@ -1,133 +1,142 @@
...
+                        in non diam. Phasellus eros lectus, tristique vitae 
+                        pellentesque ut, rhoncus vel velit. Donec rutrum 
+                        ante ac velit porttitor placerat non non tortor. 
+                        Sed tempor risus in quam blandit pharetra sit amet 
+                        eget libero. Nunc ullamcorper, augue ac auctor
+                        facilisis, augue metus ultricies ipsum, quis 
+                        ullamcorper lorem lectus id velit. Nulla eu enim purus.
+                    </p>
+                </div>
+                <div class="Title-Location">
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html]: Please use only lower case to name clases (Ex. `title-location`). Also try to always use English to name classes and id's

File: Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html [hunk 3]
```diff
@@ -1,133 +1,142 @@
...
-                <form>
-                    <label> 
-                        Name: <input type="text" name="Name" id="Name"> 
-                    </label>
-                </form>
-                <form>
-                    <label> 
-                        Email: <input type="text" name="Email" id="Email"> 
+        <div class="Container">
+                <div class="Wrapper">
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html]: In this exercise, it's required to use Bootstrap's grid system

**Review by AlexHenkel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**